### PR TITLE
Feature/update cardinfo with favorite

### PR DESCRIPTION
**Pull request by mrSelieznov**
#217 Task

Added favorite button to info card

### CODE REVIEW

**Review by solomiia-sahala — APPROVED** [review]

**Review by Dmytro1991ua — APPROVED** [review]

File: src/styles/RenderResults.scss [hunk 1]
```diff
@@ -46,3 +57,5 @@
 .small-card-wrapper{
   margin: 15px;
 }
+
+
```
Comment (oleksandrstarov) [inline src/styles/RenderResults.scss:61]: Remove empty line

File: src/styles/CardInfo.scss [hunk 2]
```diff
@@ -165,4 +165,11 @@ $white:  #ffff;
   }
 }
 
+.info-card-activities-mobile{
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:168]: add space before {

File: src/components/RenderResults.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ const RenderResults = ({ list }: RenderResultsArrList) => {
 
   return (
     <div className="delivery">
-      <div className="view-toggler">
+      <div className="view-toggler ">
```
Comment (oleksandrstarov) [inline src/components/RenderResults.tsx:23]: remove space

**Review by oleksandrstarov — CHANGES_REQUESTED** [review]
Would be nice is you pay a little more attention while creating PR

File: src/styles/CardInfo.scss [hunk 4]
```diff
@@ -1,170 +1,174 @@
...
-$orange:  #FCAE30;
-$white:  #ffff;
+$orange: #FCAE30;
+$white: #ffff;
 
 %pos-absolute-left0 {
-  position: absolute;
-  left: 0;
+ position: absolute;
+ left: 0;
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:7]: please check all your changes for styles and add indent inside blocks

File: src/styles/CardInfo.scss [hunk 5]
```diff
@@ -1,170 +1,174 @@
...
-  justify-content: center;
-  padding: 40px 20px 10px;
-  margin: 10px 10px 45px;
-  box-shadow: 0 0 5px rgba(0,0,0,0.5);
+ max-width: 280px;
+ max-height: 475px;
+ }
+
+.card-info {
+ background-color: $white;
```
Comment (oleksandrstarov) [inline src/styles/CardInfo.scss:21]: should be 2 spaces

**Review by oleksandrstarov — APPROVED** [review]

**Pull request merged**